src/olympia/blocklist/mlbf.py

```diff
@@ -233,16 +262,17 @@ def generate_and_write_filter(self):
 
         log.info(json.dumps(stats))
 
+    @lru_cache(maxsize=128)  # noqa: B019
```

Comment:
nit: I don't think we're ever going to call the function 128 times 🙃 - it's typically like, 3 times, isn't it?

Reply:
I think it can be more than three but yeah probably not 128. I think I copy pasted that from docs.. maybe there is a sensible default.